CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ Unreleased` header.
 - **Breaking:** Rename `VideoMode` to `VideoModeHandle` to represent that it doesn't hold static data.
 - **Breaking:** No longer export `platform::x11::XNotSupported`.
 - **Breaking:** Renamed `platform::x11::XWindowType` to `platform::x11::WindowType`.
+- On Redox, Update `redox_syscall` from 0.3 to 0.4.1
+- On Redox, Update `orbclient` from 0.3.42 to 0.3.47
```

Comment:
If this isn't exposed to the user, it doesn't need a changelog entry.

Reply:
Should i remove it?